docs/streams/tutorial.html

```diff
@@ -460,7 +461,7 @@
         source.flatMapValues(new ValueMapper&lt;String, Iterable&lt;String&gt;&gt;() {
                     @Override
                     public Iterable&lt;String&gt; apply(String value) {
-                        return Arrays.asList(value.toLowerCase(Locale.getDefault()).split("\\W+"));
+                        return Arrays.asList(value.toLowerCase(Locale.getDefault()));
```

Comment:
This is correct as-is. We need to split words. Same below. The different to https://github.com/apache/kafka/pull/3883/files#diff-607b296d195577513e0e0833269d4175R426 as pointed out the the paragraph above is only, that we convert to lover case before splitting.

Reply:
@mjsax Of course, you are right. The WordCount doesn't chain on the LineSplit so it still need to be there. I fixed it - should be OK now.